examples/js/loaders/GLTFLoader.js

```diff
@@ -27,6 +27,27 @@ THREE.GLTFLoader = ( function () {
 
 			var path = this.path !== undefined ? this.path : THREE.LoaderUtils.extractUrlBase( url );
 
+			var itemGroup = '[group: ' + url + ']';
+
+			scope.manager.itemStart( itemGroup );
+
+			var _onError = function ( e ) {
+
+				if ( onError ) {
+
+					onError( e );
+
+				} else {
+
+					console.error( e );
```

Comment:
Currently we throw error if `onError` is undefined. Is changing to console error for calling `manager.itemEnd/Error()` after completing error handling?

Reply:
>  This error can't be caught by developer code

Yeah, right. I don't remember why we chose to throw error... So I think console error may be ok.